lib/bootstrap_form/components/validation.rb

```diff
@@ -65,16 +65,20 @@ def generate_error(name)
         content_tag(help_tag, help_text, class: help_klass)
       end
 
+      # rubocop:disable Metrics/AbcSize
       def get_error_messages(name)
-        messages = object.errors[name]
         object.class.try(:reflections)&.each do |association_name, a|
           next unless a.is_a?(ActiveRecord::Reflection::BelongsToReflection)
           next unless a.foreign_key == name.to_s
 
-          messages << object.errors[association_name]
+          object.errors[association_name].each do |error|
+            object.errors.add(name, error)
```

Comment:
I was intrigued by your comment that it didn't say anything at version 7. You would think it should have errored, if the deprecation warning was in 6.1. You're right. `<<` seems to still work, and without a deprecation message. Very strange.

I was unable to get it to show me the deprecation message, even with version 6.1. (I tweaked the code to run your test case, but with the `<<` operator instead of `#add`.) Also strange. At any time, did you see the deprecation warning when running the test you added?

Reply:
If i revert my changes to `get_error_messages` but leave the new testcase in place i get the following:

![Screenshot 2023-08-10 at 09 28 03](https://github.com/bootstrap-ruby/bootstrap_form/assets/66613/5e8a293f-d556-482d-9262-a79025c269c2)